src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/Spinner.xaml.cs

```diff
@@ -3,21 +3,35 @@
 
 using System;
 using System.Collections.ObjectModel;
+using System.ComponentModel;
 using System.Globalization;
 using System.Linq;
+using System.Runtime.CompilerServices;
 using System.Windows.Controls;
 using System.Windows.Data;
 using System.Windows.Media;
 
 namespace NuGet.PackageManagement.UI
 {
-    internal class EllipseDetails
+    internal class EllipseDetails : INotifyPropertyChanged
     {
         public double Width { get; set; }
         public double Height { get; set; }
         public double Left { get; set; }
         public double Top { get; set; }
         public Brush Fill { get; set; }
+        
+        // Added INotifyPropertyChange to avoid memory leak in Spinner control. 
+        // Yes, we don't actually call it/use it. But it fixes the leak.
```

Comment:
Remove second line of this comment. We do not use it directly, but runtime uses it 🙂

Reply:
Ok.